lib/trust_is_risk.js

```diff
@@ -0,0 +1,202 @@
+//      
+var bcoin = require('bcoin');
+var assert = require('assert');
+var Address = bcoin.primitives.Address;
+
+                      // base58 bitcoin address
+                    
+                
+             
+                 
+                 
+                                                               
+ 
+                               
+                     
+
+class TrustIsRisk {
+                       
+  
+                 
+                        
+                            
+      
+   
+
+               
+                                 
+   
+
+  constructor(node                 ) {
+    this.node = node;
+    this.directTrust = {};
+    this.TXToTrust = {};
```

Comment:
There's type declarations in the source file under src/, which describe what types these will hold. If you still think that something is unclear and requires a comment, please comment on the specific member variable.